### ISSUE TITLE
.NET Core Support

### ISSUE THREAD

**Issue by RonnChyran** [unosquare/embedio#41]
Does EmbedIO have support for .NET Core 1.0 yet?

**Comment by mariodivece** [comment]
No not yet.. We are waiting on someone to port the HttpListener class to .NET Core. There are some initial attempts already (https://www.google.com.mx/?gfe_rd=cr&ei=HBylV_uzO8ra8geaz7fgAg#q=.net+core+httplistener). If you'd like to try them out please let us know your results and suggestions.

**Comment by geoperez** [comment]
Another HttpListener Core implementation: https://github.com/robertsundstrom/HttpListener

**Comment by falahati** [comment]
Well, I wanted to use this project on a Universal Window 10 app and failed for the same reason. Please port it to .Net Core.
BTW, robertsundstrom's implementation of HttpListener seems promising but still lacks some features. I don't know if your project uses these features tho.

Thanks for the great project.

**Comment by geoperez** [comment]
I'm working on a HTTPListener port using as base the MONO project. I'll update this ticket when a stable version is released.

So far I'm able to use the WebAPI and StaticFiles. But SSL and WebSockets are out of scope so far.

**Comment by falahati** [comment]
Just a suggestion,
https://github.com/rdavisau/sockets-for-pcl

**Comment by geoperez** [comment]
The first Nuget with support to .NET Core was released. Version 1.1.0.
